.babelrc

```diff
@@ -1,21 +1,20 @@
 {
   "presets": [
-    "flow",
-    ["env", { "modules": false, "loose": true }]
+    "@babel/preset-flow",
+    ["@babel/preset-env", {
+      "loose": true
+    }]
   ],
   "comments": false,
-  "plugins": [
-    "transform-object-rest-spread"
-  ],
   "env": {
     "cjs": {
       "plugins": [
-        "transform-es2015-modules-commonjs"
+        "@babel/plugin-transform-modules-commonjs"
       ]
     },
     "test": {
       "plugins": [
-        "transform-es2015-modules-commonjs"
+        "@babel/plugin-transform-modules-commonjs"
```

Comment:
this env section with `transform-modules-commonjs` can be removed since env is already includes this. The point is that rollup/webpack plugins are able to add `modules` option internally. This allows to simplify config.

Reply:
I will remove this

Reply:
I’ll do it tomorrow